internal/scripts/installation/compile_MET_all.sh

```diff
@@ -442,6 +434,23 @@ fi
 # Compile Proj
 if [ $COMPILE_PROJ -eq 1 ]; then
 
+
+  if [ $COMPILE_TIFF -eq 1 ]; then
+    echo
+    echo "Compiling TIFF at `date`"
+    mkdir -p ${LIB_DIR}/tiff
+    rm -rf ${LIB_DIR}/tiff/tiff*
+    tar -xzf ${TAR_DIR}/tiff*.tar.gz -C ${LIB_DIR}/tiff
+    cd ${LIB_DIR}/tiff/tiff*
+    echo "cd `pwd`"
+    #run_cmd "./configure --prefix=${LIB_DIR} LDFLAGS=-L${LIB_DIR}/lib CPPFLAGS=-I${LIB_DIR}/include > $(pwd)/tiff.configure.log 2>&1"
```

Comment:
@jprestop can this commented out line be removed or should it remain for some reason?
```suggestion
```

Reply:
Thanks for catching that commented out line.  It should be removed as you suggested.